test/mocha/10-api.js

```diff
@@ -281,6 +524,49 @@ describe('bedrock-profile-http', () => {
       result.ok.should.equal(true);
       result0.data.profileAgent.zcaps.should.eql(zcaps);
     });
+    it('throws error when there is no zcaps', async () => {
+      const {account: {id: account}} = accounts['alpha@example.com'];
+      const noZcaps = '';
+      let result;
+      let error;
+      try {
+        const {data: {id: profile}} = await api.post('/profiles', {account});
+        const {data} = await api.get(`/profile-agents/?account=${account}` +
+          `&profile=${profile}`);
+        const {id: profileAgentId} = data[0].profileAgent;
+        result = await api.post(`/profile-agents/${profileAgentId}` +
+          `/capability-set?account=${account}`, {zcaps: noZcaps});
+      } catch(e) {
```

Comment:
Too much stuff in this try.  Only put the API calls under test in the try/catch so that the assertions only apply to that thing.  Otherwise if any of these other things throw for some other reason, the assertions will fail, but you won't know which thing failed.

Reply:
This applies throughout.

Reply:
Understood, I removed everything except the API calls from the `try` block.